scripts/appleseedMaya/renderGlobals.py

```diff
@@ -483,7 +483,7 @@ def create(self):
                             ui=pm.intSliderGrp(
                                 label="Max Samples",
                                 field=True,
-                                value=256,
+                                value=128,
```

Comment:
It's 256 in appleseed.studio, and IIRC a recent commit of yours aligned appleseed-maya with appleseed.studio, so why this change?

Reply:
In Maya the default Max value serves also as the default sample value for uniform sampler. If one uses the uniform sampler default 256 samples is a bit too high. In studio default for uniform is 64, so 128 is a compromise to not have too low default Max samples when the adaptive sampler is chosen.
I guess @est77 preferred the simpler UI-layout without the need to add an additional control for uniform samples.

Reply:
Ah ok, makes sense then.